entgql/extension.go

```diff
@@ -215,7 +269,7 @@ func (e *Extension) mapScalar(f *gen.Field, op gen.Op) string {
 		scalar = graphql.Boolean.Name()
 	case f.IsEdgeField():
 		scalar = graphql.ID.Name()
-	case t.Numeric():
+	case t.Integer():
```

Comment:
Nice catch, This is a pretty nasty bug you should probably create a seperate pr for

Reply:
I also bumped into this a few days ago, https://github.com/ent/contrib/pull/114/files